drivers/include/at86rf2xx.h

```diff
@@ -37,6 +37,7 @@
 #include "periph/gpio.h"
 #include "net/netdev.h"
 #include "net/netdev/ieee802154.h"
+#include "ieee802154_config.h"
```

Comment:
wouldn't it be more in the spirit of this PR to introduce a `at86rf2xx_config.h` to set all the values below, and include that here and use `ieee802154_config.h` only in that new header file?

Reply:
I like the idea!